.changelogs/6303.json

```diff
@@ -0,0 +1,7 @@
+{
+  "title": "Improved the performance of the `getCellMeta` method",
+  "type": "changed",
+  "issue": 6303,
+  "breaking": true,
```

Comment:
Performance is not a breaking change. Maybe we should add second changelog file `6303-hooks.json` that will describe the breaking change precisely? Currently, with "Improved the performance of the `getCellMeta` method" we're missing info that hooks have changed

Reply:
Good idea 👍  d75093324a59bb942a80af10a33b637e50dd46d1